src/Classes/Cli/TextRenderer.php

```diff
@@ -24,50 +24,39 @@ public static function color(string $text, int $color): string
         return "\e[" . $color . "m" . $text . "\e[0m";
     }
 
-    public static function rightPad(string $text, int $totalLength): string
+    public static function rightPad(string $text, int $length): string
     {
-        $textLength = strlen($text);
-
-        if ($textLength >= $totalLength) {
-            return $text;
-        }
-
-        $paddingLength = $totalLength - $textLength;
-        $padding = str_repeat(' ', $paddingLength);
-
-        return $text . $padding;
+        return \str_pad($text, $length, ' ', \STR_PAD_RIGHT);
     }
 
-    public static function renderHelpHeading(string $heading): string
+    public static function renderLogo()
     {
-        return self::color($heading, self::COLOR_YELLOW) . "\n";
+        // echo "    __  _____  _____    ______   ________    ____\n";
+        // echo "   /  |/  /  |/  / /   / ____/  / ____/ /   /  _/\n";
+        // echo "  / /|_/ / /|_/ / /   / /      / /   / /    / /  \n";
+        // echo " / /  / / /  / / /___/ /___   / /___/ /____/ /   \n";
+        // echo "/_/  /_/_/  /_/_____/\____/   \____/_____/___/   \n";
+        // created with: https://patorjk.com/software/taag/#p=display&f=Slant&t=MMLC%20CLI
+
+        echo "    __  ___ __  ___ __    ______   ______ __     ____\n";
+        echo "   /  |/  //  |/  // /   / ____/  / ____// /    /  _/\n";
+        echo "  / /|_/ // /|_/ // /   / /      / /    / /     / /  \n";
+        echo " / /  / // /  / // /___/ /___   / /___ / /___ _/ /   \n";
+        echo "/_/  /_//_/  /_//_____/\____/   \____//_____//___/   \n";
+        // cretated with: https://patorjk.com/software/taag/#p=display&h=1&f=Slant&t=MMLC%20CLI
     }
 
-    public static function renderHelpCommand(string $name, string $description, int $pad = 20): string
+    public static function getPadding(array $items): int
     {
-        $name = self::rightPad($name, $pad);
-        return "  " . self::color($name, self::COLOR_GREEN) . " $description\n";
-    }
+        $padding = 0;
 
-    public static function renderHelpArgument(string $name, string $description, int $pad = 20): string
-    {
-        $name = self::rightPad($name, $pad);
-        return "  " . self::color($name, self::COLOR_GREEN) . " $description\n";
-    }
-
-    public static function renderHelpOption(string $shortName, string $longName, string $description, int $pad = 20): string
-    {
-        $name = '';
-
-        if ($shortName && $longName) {
-            $name = "-$shortName, --$longName";
-        } elseif ($shortName) {
-            $name = "-$shortName";
-        } elseif ($longName) {
-            $name = "    --$longName";
+        foreach ($items as $item) {
+            $itemLength = \mb_strlen($item);
+            $padding = \max($padding, $itemLength);
         }
 
-        $name = self::rightPad($name, $pad);
-        return "  " . self::color($name, self::COLOR_GREEN) . " $description\n";
+        $padding += 1;
```

Comment:
I think we shouldn't calculate +1 at this point because it distorts the result. The caller can decide for themselves how much padding they want. Maybe he doesn't want space.